plugins/modules/make.py

```diff
@@ -89,6 +89,12 @@
     - The value of the module parameter I(chdir).
   type: str
   returned: success
+command:
+  description:
+    - The command built and executed by the module
```

Comment:
Sorry, I missed this earlier:
```suggestion
    - The command built and executed by the module.
```

Reply:
also added missing dots to some of the other entries.